frontends/web/src/api/bitbox02bootloader.ts

```diff
@@ -31,6 +31,14 @@ export const getStatus = (
   return apiGet(`devices/bitbox02-bootloader/${deviceID}/status`);
 };
 
+export const syncStatus = (deviceID: string) => {
+  return (
+    cb: TSubscriptionCallback<TStatus>
+  ) => {
+    return subscribeEndpoint(`devices/bitbox02-bootloader/${deviceID}/status`, cb);
+  };
```

Comment:
the get/sync pair used in `useSync()` is most often at the same endpoint, isn't it? sounds like we need another hook that only takes the endpoint and the object type and automatically calls `apiGet()` and `subscribeEndpoint()` :grin:

Reply:
yeah it must subscribe at the same endpoint otherwise this wont reload 

https://github.com/digitalbitbox/bitbox-wallet-app/blob/56d4eab5dcd46fec73ec2f3675a7bb00c945b698/frontends/web/src/api/subscribe.ts#L40

there is only a few using action.reload, maybe we can migrate those to action.replace and get rid of reload all together in the future?

![Screen Shot 2023-12-29 at 13 38 31](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/eecc2ac4-6624-4d1e-a422-1196039ef778)

Reply:
In theory that should be possible, but not sure if worth it? The only reason to do it would be that it would be faster, right? But I doubt it would be noticeable.